drivers/fuel_gauge/sbs_gauge/sbs_gauge.c

```diff
@@ -95,9 +95,9 @@ static int sbs_gauge_get_prop(const struct device *dev, struct fuel_gauge_get_pr
 		rc = sbs_cmd_reg_read(dev, SBS_GAUGE_CMD_VOLTAGE, &val);
 		prop->value.voltage = val * 1000;
 		break;
-	case FUEL_GAUGE_MODE:
```

Comment:
Let's do renaming in a separate PR. I'm happy to care of that too since

Reply:
I renamed only at rate properties in a separate commit in this PR as it needs to be done anyways